Automattic/jetpack
# Add stat trends for videopress my jetpack card

## Proposed changes:

Add stat trends for the value indicators on the VideoPress card in My Jetpack

### Other information:


## Jetpack product discussion

P2: p1HpG7-rb7-p2

## Does this pull request change what data or activity we track or use?

Yes, a prop for previous stats is added to the tracks that are tracked when the tooltip is clicked

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. Make sure your site has at least one video
3. To mock some randomized data, go to fbhepr%2Skers%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sjcpbz%2Qwfba%2Qraqcbvagf%2Spynff.jcpbz%2Qwfba%2Qncv%2Qfgngf%2Qivqrb%2Qcynlf%2Qi1%2Q1%2Qraqcbvag.cuc%3Se%3Q210o65nn%26zb%3Q1199%26sv%3Q37%2350-og and replace all those zeros with something like `rand(0, 100)`. You can pick whatever numbers you'd like if you wanted to test out smaller or bigger numbers.
4. Go to My Jetpack and make sure the stat trends are shown and look good on all screen sizes
![image](https://github.com/user-attachments/assets/79308bb9-ef9d-4ff3-9872-d00e6f0cd979)
